src/transformations/transform-constructor.ts

```diff
@@ -70,6 +73,9 @@ export function* removeLeftoverConstructorHead(sourceUnit: SourceUnit): Generato
   }
 }
 
+// Inserts the init and unchained function declarations before the constructor first `{`, for example
+// This: constructor(/* arguments */) /* modifiers */ public
+// Results in: constructor(/* arguments */) /* modifiers */ public { function __Name_init(/*arguments*/) /* modifiers */
```

Comment:
Since this is an example, let's use actual arguments instead of `/* arguments */`. Same for modifiers, etc.